puppet/modules/web/files/rsync_downloads.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+set -e
+
+case "$SSH_ORIGINAL_COMMAND" in
+*\&*)
+echo "Rejected"
+;;
+*\(*)
+echo "Rejected"
+;;
+*\{*)
+echo "Rejected"
+;;
+*\;*)
+echo "Rejected"
+;;
+*\<*)
+echo "Rejected"
+;;
+*\`*)
+echo "Rejected"
+;;
+*\|*)
+echo "Rejected"
+;;
+rsync\ --server*)
+  # Permit transfer
+  $SSH_ORIGINAL_COMMAND
+;;
+*)
+echo "Rejected"
+;;
+esac
+# ERB highlighting looks terrible in this script...
+# vim: set ft=sh : #
```

Comment:
Is this really still needed? Now that it's in files, it doesn't have an ERB extension.

Reply:
yeah, copy paste error